src/main/java/ca/corefacility/bioinformatics/irida/ria/web/activities/dto/Activity.java

```diff
@@ -23,6 +23,14 @@ public Activity(Long id, String type, String description, Date date, List<Activi
 		this.items = items;
 	}
 
+	public Long getId() {
```

Comment:
Added in getters for attributes that were otherwise triggering `unused` compiler warnings.